01-starting-with-data.Rmd

```diff
@@ -356,6 +356,7 @@ The maximum value rises and falls perfectly smoothly, while the minimum seems to
 
 #### Key Points
 
+* 
```

Comment:
Actually, if you look through the pull requests of that file, I've added a new key point.